php/class-delivery.php

```diff
@@ -897,8 +897,13 @@ public function convert_tags( $content, $context = 'view' ) {
 			$base                   = $type . ':' . $url;
 			$public_id              = ! is_admin() ? $relation['public_id'] . '.' . $relation['format'] : null;
 			$cloudinary_url         = $this->media->cloudinary_url( $relation['post_id'], array(), $relation['transformations'], $public_id );
-			$aliases[ $base . '?' ] = $cloudinary_url . '&';
-			$aliases[ $base ]       = $cloudinary_url;
+			if ( ! empty( $relation['slashed'] ) && $relation['slashed'] ) {
+				$aliases[ $base . '?' ] = trim( wp_json_encode( $cloudinary_url . '&' ), '"' );
+				$aliases[ $base ]       = trim( wp_json_encode( $cloudinary_url ), '"' );
```

Comment:
@pereirinha - Would be cleaner to do:
```suggestion
				$aliases[ $base . '?' ] = addcslashes( $cloudinary_url . '&', '/' );
				$aliases[ $base ]       = addcslashes( $cloudinary_url ), '/' );
```